From: Shannon Nelson <snelson@pensando.io>
Date: Thu, 17 Sep 2020 18:13:25 -0700
Subject: netdevsim: devlink flash timeout message
Patch-mainline: v5.10-rc1
Git-commit: b311b001dea14555753c6c4f1f089689a18413bb
References: bsc#1176447

Add a simple devlink flash timeout message to exercise
the message mechanism.

Signed-off-by: Shannon Nelson <snelson@pensando.io>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/netdevsim/dev.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/net/netdevsim/dev.c
+++ b/drivers/net/netdevsim/dev.c
@@ -767,6 +767,8 @@ static int nsim_dev_flash_update(struct
 						   component,
 						   NSIM_DEV_FLASH_SIZE,
 						   NSIM_DEV_FLASH_SIZE);
+		devlink_flash_update_timeout_notify(devlink, "Flash select",
+						    component, 81);
 		devlink_flash_update_status_notify(devlink, "Flashing done",
 						   component, 0, 0);
 		devlink_flash_update_end_notify(devlink);
